src/service/Github.ts

```diff
@@ -230,6 +230,38 @@ export class GithubService {
     return result.data;
   }
 
+  async removeFilesFromPR(
+    files: { path: string; content?: string; }[],
+    sourceBranch, baseSha,
+    originBranch?: string
+  ) {
+
+    console.log("asdf");
+
+    originBranch = originBranch ? originBranch : "master";
+    console.log(`originBranch = ${originBranch}`)
+
+    const paths = files.filter(file => !file.content).map(file => file.path);
+    console.log(`paths = ${paths}`)
+    let blobs = await this.getFilesFromTree(paths, originBranch);
+    console.log(`blobs = ${blobs}`)
+    blobs = blobs.map(({ path, mode, type, sha }) => ({ path, mode, type, sha }));
+    console.log(`blobs = ${blobs}`)
+    for (let i = 0; i < paths.length; i++) {
+      try {
+        const path = paths[i];
+        const fileFromMaster = await this.getFile(path, originBranch);
+        blobs.push({ path, mode: "100644", type: "blob", sha: fileFromMaster.sha })
```

Comment:
The type and mode should come from fileFromMaster blob info, because it is not always 100644

Reply:
Never mind